esp/services/ws_machine/ws_machineService.cpp

```diff
@@ -2808,8 +2796,6 @@ void Cws_machineEx::readComponentUsageResult(IEspContext& context, IPropertyTree
                             VStringBuffer desc("%s not found. Read disk usage from %s", aDiskPath, pathUsed);
                             diskUsage->setDescription(desc);
                         }
-                        if ((version >= 1.18) && folderTree->hasProp("@size"))
```

Comment:
Is there any compatibility issue if you remove this ?  Is it safer to just leave the third argument but have it be "0" for now ?

Reply:
The 2 lines of the code sets the ActualSIze only if the folerTree has the '@size'. Theoretically, the client side should not have a problem if the client does not receive the ActualSIze. So, I think it should no compatibility issue here.